lib/sgtn-client/api/base.rb

```diff
@@ -0,0 +1,92 @@
+# 
+#  Copyright 2019-2022 VMware, Inc.
+#  SPDX-License-Identifier: EPL-2.0
```

Comment:
Please format this file because it looks so messy.

Reply:
Let's have another PR for the clean up.

Reply:
Do the whole moving thing in another PR.

Reply:
Do you guys know what's the formatter we should use and how to use it? It loos fine on my IDE.

Reply:
@jessiejuachon, the move is not cleanup, it's necessary to put them in the same file.